test/test-threadpool-cancel.c

```diff
@@ -87,8 +87,32 @@ static void unblock_threadpool(void) {
 }
 
 
+static int known_broken(uv_req_t* req) {
+  if (req->type != UV_FS)
+    return 0;
+
+#ifdef __linux__
+  /* TODO(bnoordhuis) make cancellation work with io_uring */
```

Comment:
Is it `IORING_OP_ASYNC_CANCEL` worth trying or as it doesn't work for disk IO requests it will never work?

Reply:
I opted to leave out cancellation for now. I do think it can be made to work but that's for another day (and pull request.)

Reply:
Got it. Are we ok leaving this support out without having linux users a "official" way to opt-out of iouring?

Reply:
I'd argue it's not really a change in behavior because a libuv user has to handle UV_EBUSY from `uv_cancel()` anyway. I mean, we could change it to this:
```c
int uv_cancel(uv_req_t* req) { return UV_EBUSY; }
```
And it'd still be _pro forma_ backwards compatible. :-)